config.py

```diff
@@ -1,6 +1,6 @@
-import os
 
-chromedriver_path = os.path.dirname(
-    os.path.realpath(__file__)) + '/resources/chromedriver.exe'
+
+win_chromedriver_path = '/resources/chromedriver.exe'
+mac_chromedriver_path = '/resources/chromedriver'
 
```

Comment:
All I have is questions about how some of this works. I didn't see any issues with the code.